.dockerfile

```diff
@@ -0,0 +1,6 @@
+*
+!src
+!views
```

Comment:
If we're including the src directory and all its contents, do we need !views? Plus, I'm wondering if this should be a .dockerignore file, I'll get some clarity on this and get back to you.

Reply:
yes i think you are right, the contents looks like a .dockerignore file, so we'll need one of those and delete the .dockerfile , I don't think it's needed as we have Dockerfile

Reply:
I have deleted the .dockerfile file and created the .dockerignore file.

Reply:
@makhtar-ch  did you get any additional clarity?

Reply:
yeah Mark pretty much covered it above.